### ISSUE TITLE
Switching in left Nav

### ISSUE THREAD

**Issue by flyingUnderTheRadar** [Microsoft/accessibility-insights-windows#1]
When in live inspect / events I cannot switch to "Test" mode unless I go back to live inspect mode first. (Can still go to color contrast test)
